packages/algo-ts/src/arc4/encoded-types.ts

```diff
@@ -265,14 +265,18 @@ export class Address extends Arc4ReadonlyArray<Byte> {
   get native(): Account {
     throw new NoImplementation()
   }
+
+  copy(): Address {
+    throw new Error('Method not implemented.')
+  }
 }
 
-type StructConstraint = Record<string, ARC4Encoded>
+export type StructConstraint = Record<string, ARC4Encoded>
 
 class StructBase extends ARC4Encoded {
   [TypeProperty] = 'arc4.Struct'
 }
-class StructImpl<T extends StructConstraint> extends StructBase {
+export class StructImpl<T extends StructConstraint> extends StructBase {
```

Comment:
We currently re-export all exported types from this file in `./arc4/index.ts`. I don't think we want this type (and others) to end up in the non-internal exports so we should replace the `export * from './encoded-types.ts'` with `export { Str, Uint..... } from './encoded-types'` (ie. only include the types an algo-ts consumer should actually use

Reply:
It has been reverted.